buildenv/Build_Instructions_V8.md

```diff
@@ -0,0 +1,322 @@
+<!--
+Copyright (c) 2017, 2017 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code is also Distributed under one or more Secondary Licenses,
+as those terms are defined by the Eclipse Public License, v. 2.0: GNU
+General Public License, version 2 with the GNU Classpath Exception [1]
+and GNU General Public License, version 2 with the OpenJDK Assembly
+Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+-->
+
+Building OpenJDK Version 8 with OpenJ9
+======================================
+
+Our website describes a simple [build process](http://www.eclipse.org/openj9/oj9_build.html)
```

Comment:
This describes the Java 9 simple build process.  Is there an equivalent page for java 8?

Reply:
It's the same page for both versions. There is a toggle to flip between instructions for 8 or 9.